Rework background and (non) goals

felixlinker opened the pull request:
Digest of changes:
- Editing of background
- New section "goals" that defines (in a minimal sense) what we would like to work on, i.e., things that are "in-scope"
- Renamed "Out of scope" to "Non-Goals" and moved it after goals

felixlinker committed:
Rework background and (non) goals

felixlinker commented:
@mstojens Maybe you want to pick up here when you work on the charter tomorrow! In any case, I'm curious about your feedback.

tojens-ietf commented on authenticated-digital-emblem-scoping.md:
```diff
@@ -2,7 +2,27 @@
 
-During armed conflicts, infrastructure marked with certain distinctive emblems or signs, including the Red Cross, Red Crescent, and Red Crystal, are given special protections under International Humanitarian Law (IHL) and cannot be attacked under almost all circumstances. As conflicts now involve cyberattacks against digital resources, clear authenticated communication of equivalent digital emble ...
+During armed conflict, certain humanitarian infrastructure enjoys special protection under International Humanitarian Law (IHL) and must be respected and protected by all parties to the conflict.
+IHL defines a number of physical emblems, including the Red Cross, Red Crescent, and Red Crystal, that signal this protection in the physical world.
```
```suggestion
IHL defines a number of physical emblems, including the Red Cross, Red Crescent, and Red Crystal, that visually signal this protection for physical assets using simple representations that enable the easy adoption and recognition needed for these emblems to be widely adopted in real conflict environments.
```

To explain why the current emblems are not insufficient for the physical asset use case and set the stage for arguments that matrix "discovery channel" against "attack channel".

tojens-ietf commented on authenticated-digital-emblem-scoping.md:
```diff
@@ -2,7 +2,27 @@
-During armed conflicts, infrastructure marked with certain distinctive emblems or signs, including the Red Cross, Red Crescent, and Red Crystal, are given special protections under International Humanitarian Law (IHL) and cannot be attacked under almost all circumstances. As conflicts now involve cyberattacks against digital resources, clear authenticated communication of equivalent digital emble ...
+During armed conflict, certain humanitarian infrastructure enjoys special protection under International Humanitarian Law (IHL) and must be respected and protected by all parties to the conflict.
+IHL defines a number of physical emblems, including the Red Cross, Red Crescent, and Red Crystal, that signal this protection in the physical world.
+However, there is no way to signal this protection in the digital world.
```
```suggestion
However, there is no way to visually signal this protection when the emblem-bearing asset is a digital concept and therefore has no equivalent physical form to which a visual representation of an emblem can be applied. Such digital assets are both inspectable and attackable without kinetic means or ever seeing the visual form of whatever real hardware is running the digital asset. 
```

To highlight why digital enhancement of physical emblems is not sufficient even ignoring the effect it would have on the under-privileged (that cyberattacks never see a server rack visually the way a bomber can see a building physically since determining the mapping between a digital asset and its physical location is virtually impossible to guarantee).

tojens-ietf commented on authenticated-digital-emblem-scoping.md:
```diff
@@ -2,7 +2,27 @@
+However, there is no way to signal this protection in the digital world.
+
+As conflicts now increasingly take place in the digital world, signalling protection using equivalent digital emblems facilitates the protection of medical and humanitarian digital assets.
+Digital emblems must be accessible through well-known mechanisms and must be respected, verifiable, and accountable against misuse.
```
```suggestion
Digital emblems must be accessible through well-known mechanisms and must be recognizable, verifiable, and accountable against misuse to enable good-faith actors to respect the meaning of the emblem.
```

To avoid implications of defining anything that is capable of coercing behavior (forcing the observer to respect the emblem, which we cannot do).

tojens-ietf commented on authenticated-digital-emblem-scoping.md:
```diff
@@ -2,7 +2,27 @@
 ## Background
 
-During armed conflicts, infrastructure marked with certain distinctive emblems or signs, including the Red Cross, Red Crescent, and Red Crystal, are given special protections under International Humanitarian Law (IHL) and cannot be attacked under almost all circumstances. As conflicts now involve cyberattacks against digital resources, clear authenticated communication of equivalent digital emble ...
+During armed conflict, certain humanitarian infrastructure enjoys special protection under International Humanitarian Law (IHL) and must be respected and protected by all parties to the conflict.
```
```suggestion
During armed conflict, certain humanitarian infrastructure enjoys special protection under International Humanitarian Law (IHL) which mandates that such infrastructure must be respected and protected by all parties to the conflict.
```

To clarify the "must" as a legal MUST and not a proposed RFC MUST.

tojens-ietf commented on authenticated-digital-emblem-scoping.md:
```diff
@@ -2,7 +2,27 @@
+
+The working seeks to develop a mechanism that:
+
+- can signal that a digital asset enjoys protection under IHL using network interfaces;
```
```suggestion
- can signal that a digital asset enjoys protection under IHL using network interfaces (so that targets attackable by network communication can also present emblems through network communication);
```

Adding justification for the requirement in-line for clarity

tojens-ietf commented on authenticated-digital-emblem-scoping.md:
```diff
@@ -2,7 +2,27 @@
+The working seeks to develop a mechanism that:
+
+- can signal that a digital asset enjoys protection under IHL using network interfaces;
+- parties to a conflict are willing to use; such parties typically intend to engage in offensive operations and, for example, will not tolerate signalling this intention.
```
```suggestion
- parties to a conflict are willing to use; such parties typically intend to engage in offensive operations and, for example, will not tolerate signalling this intention.
```

I would want to add this before the semicolon, but I did not follow your meaning in the sentence after your semicolon. Can you clarify for me please?

"(because digital representations of these emblems must be officially recognized under IHL)"

felixlinker replied:
Sentence after semicolon = covert inspection (but generalized). I, on the other hand, do not quite follow what you want to say with the addition in parentheses 😁 Could you elaborate? I think the "willing to use" part is not just for compliance with existing IHL. Actually, your addition might warrant its own bullet point.

felixlinker replied:
But I certainly agree that my sentence after semicolon is quite vaguely put.

tojens-ietf replied:
Ohh, I see, I was confused by signal meaning "warn the target you are verifying they do not have an emblem" versus "informing others I have an emblem". Now I see we are indeed making two different points, and I agree about splitting them. How about this (EDIT: I changed the "covert inspection" justification to align with list discussion): 

```suggestion
- parties to a conflict are willing to use (because probing for a digital emblem does not identify a cyber operator as a potential threat actor); 
- has design agreement with the necessary legal guardians of the in-scope IHL emblems (so that the working group's produced work can actually be used for its intended purpose);
```

tojens-ietf commented on authenticated-digital-emblem-scoping.md:
```diff
@@ -2,7 +2,27 @@
+  * Distinctive emblems and signs are protective because of IHL and the respect of those symbols. The group will not consider work on defining mechanisms to directly protect against cyberattack.
+
+* Deconfliction
+  * Notification or identification of physical resources or locations of people or resources to protect from physical attacks are accomplished through other methods, including physical emblems and deconfliction lines. This group will not design systems for labeling physical assets or indicating protection against physical or kinetic attack.
```
```suggestion
  * Notification or identification of physical resources or locations of people or resources to protect from physical attacks are accomplished through other methods, including physical emblems and deconfliction lines. This group will not design systems for labeling physical assets or indicating protection against physical or kinetic attack, only digital attacks (where there may be a physical and digital form for the same asset).
```

To clarify that this applies even when there's a close tie between digital form and physical form (i.e. an IoT device or IoT-enabled vehicle, not so much a server which can migrate at any moment)

tojens-ietf reviewed (commented):
Feel free to reject any or all of these and ask me to issue my own PR, but I like the structure of yours so much I figured I would just add clarifications into the same places you defined things.

felixlinker committed:
Elaborate on physical emblems

Co-authored-by: Tommy Jensen <45110146+mstojens@users.noreply.github.com>

felixlinker committed:
Update authenticated-digital-emblem-scoping.md

Co-authored-by: Tommy Jensen <45110146+mstojens@users.noreply.github.com>

felixlinker committed:
Update authenticated-digital-emblem-scoping.md

Co-authored-by: Tommy Jensen <45110146+mstojens@users.noreply.github.com>

felixlinker committed:
Apply suggestions from code review

Co-authored-by: Tommy Jensen <45110146+mstojens@users.noreply.github.com>

felixlinker committed:
Editing

felixlinker committed:
Trim trailing whitespaces

felixlinker commented:
Hey @mstojens! Thanks for your suggestions. I liked the direction but did some further editing. Here my reasoning:

- Get rid of word "attack" (Samit will want this)
- Generally shorter sentences
- Move your elaboration on physical/digital to second paragraph as this is where I introduced the concept of digital assets first (otherwise we would transition twice, breaking flow)
- Simplify the reasoning of "we cannot apply a physical emblem." I found the two sentences "we cannot apply a physical emblem because these assets have no physical form" and "we maybe could add a physical emblem to hardware" a bit contradictory. But I liked the direction so I rewrote them to "Currently, we cannot do X." This also reflects Samit's reasoning on what we're trying to do ("solve the currently impossible" as opposed to "enhance the currently possible").

Let me know what you think!

tojens-ietf commented on authenticated-digital-emblem-scoping.md:
```diff
@@ -2,27 +2,41 @@
+IHL defines a number of physical emblems, including the Red Cross, Red Crescent, and Red Crystal, which visually signal protection under IHL for physical assets.
+These physical emblems have been adopted and recognized widely in real conflict environments.
+
+Nowadays, conflicts increasingly take place in the digital world and digital assets can likewise enjoy protection under IHL.
```
```suggestion
Nowadays, conflicts increasingly take place in the digital world and digital assets also already enjoy protection under IHL.
```

Feel free to phrase better, but the point being that they DO already, not that they CAN/COULD.

felixlinker replied:
Well, I'm conflicted. On the one hand, being punchy is good, but this sounds like all digital assets are protected (they are not). But I have an idea, give me a second.

tojens-ietf commented on authenticated-digital-emblem-scoping.md:
```diff
@@ -2,27 +2,41 @@
-* accountable for misuse; 
+1. Define the requirements for authenticating, distributing, and the digital distinctive emblems and signs under IHL for network endpoints and resources. Requirements will include:
+* authenticatable by multiple, decentralized sources;
+* covert inspection;
```
You may as well reuse the agreed upon requirement re-phrasing Rohan suggested: 

```suggestion
* inspectable without identifying the inspector as a potential threat actor;
```

tojens-ietf commented:
@felixlinker thanks for the return feedback. I think everything you're saying makes sense. I've now only left:
- A revision to the open item for your review
- Rephrasing of where we refer to The Artist Formerly Known as Covert Inspection into the phrasing that had better agreement on list (generalizing it to its intended outcome)

felixlinker committed:
Apply suggestions from code review

Co-authored-by: Tommy Jensen <45110146+mstojens@users.noreply.github.com>

felixlinker committed:
Editing

felixlinker commented:
@mstojens Final editing. Super minor. I think, you'll agree. Good to go from my side!

tojens-ietf commented:
@felixlinker yes, great catch on not declaring every digital asset in the world protected xD 

LGTM, thanks for bearing with me!

felixlinker committed:
IHL fix

felixlinker commented:
Closed because it got merged into fork: https://github.com/felixlinker/emblem-scope/commit/a9d1142927dab06d73748462a7224a05e3b7e731
